src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -510,7 +519,35 @@ public static SyntaxTree ParseWithRoundTripCheck(string text, CSharpParseOptions
                 references,
                 options);
             CompilationExtensions.ValidateIOperations(createCompilationLambda);
-            return createCompilationLambda();
+            var compilation = createCompilationLambda();
+            // 'skipUsesIsNullable' may need to be set for some tests, particularly those that want to verify
```

Comment:
>// 'skipUsesIsNullable' may need to be set for some tests, particularly those that want to verify [](start = 12, length = 97)

Consider doing this on a copy, createCompilationLambda can create one.

Reply:
There were many failures when trying that. Added a `// PROTOTYPE` comment for now.

---
In reply to: [171142355](https://github.com/dotnet/roslyn/pull/25087#discussion_r171142355) [](ancestors = 171142355)